libkbfs/block_types.go

```diff
@@ -175,6 +162,11 @@ func (cb *CommonBlock) Set(other Block) {
 	cb.SetEncodedSize(otherCopy.GetEncodedSize())
 }
```

Comment:
Super nit: I can't comment there, but the `otherCopy` above is not a copy, so perhaps `otherCommon`?